### PR TITLE
fix: Steps with process style in compact theme

### LINKED ISSUE #48244: umi 开启紧凑模式，Steps组件样式有问题
### Reproduction link

[Edit on StackBlitz](https://stackblitz.com/edit/react-udqywh-aedijz?file=demo.tsx)

### Steps to reproduce

antd 使用紧凑算法，steps组件样式计算有问题。

### What is expected?

应该进度条不会偏移

### What is actually happening?

https://stackblitz.com/edit/react-udqywh-aedijz?file=demo.tsx

| Environment | Info |
| --- | --- |
| antd | 5.16.0 |
| React | ^17.x |
| System | windows |
| Browser | 谷歌 |

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

resolve #48244

### 💡 Background and solution

历史的回旋镖在 antd 支持了动态主题后打了回来。。。

截屏2024-04-03 18 03 53

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Fix Steps in compact theme not show the correct progress style.     |
| 🇨🇳 Chinese |     修复 Steps 的进度样式在紧凑模式下不正确的问题。     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — DISMISSED** [review]

File: components/steps/style/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { unit, type CSSObject } from '@ant-design/cssinjs';
 import type { CSSProperties } from 'react';
+import { unit, type CSSObject } from '@ant-design/cssinjs';
+
```
Comment (Wxh16144) [inline components/steps/style/index.ts:3]:
```suggestion
```

**Review by Wxh16144 — COMMENTED** [review]
混个 co-authored

**Review by li-jia-nan — DISMISSED** [review]

**Comment by afc163** [comment]
留一个 demo 看效果？

**Comment by zombieJ** [comment]
> 留一个 demo 看效果？

调整在 `label-placement` 里了。

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**